### PR TITLE
LCOW: Remove hard-coding

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

Removes the next `// TODO @jhowardmsft LCOW Support`... Uses the options rather than hard-coding.

@johnstep PTAL.

### CODE REVIEW

**Comment by jhowardmsft** [comment]
@darrenstahlmsft FYI. This is the one that will conflict with the options-removal-for-OCI. Could you review this quickly as well though.

**Review by johnstep — APPROVED** [review]
LGTM

**Review by darstahl — APPROVED** [review]
LGTM (not a maintainer)

**Comment by jhowardmsft** [comment]
ping @mlaventure @thaJeztah @vdemeester Been reviewed ^^. Any chance one of you can do a once over and merge/comment?

**Review by mlaventure — APPROVED** [review]
LGTM

**Pull request merged**
